gapis/api/vulkan/api/descriptor.api

```diff
@@ -803,10 +889,7 @@ sub void dovkCmdBindDescriptorSets(ref!vkCmdBindDescriptorSetsArgs args) {
               }
 
               if !hasBeenRead.b {
-                oldSets := ProcessedDescriptorSets.val[set].bufferOffsets
-                ProcessedDescriptorSets.val[set] = 
-                  ProcessedDescriptorSet(oldSets, true)
-
+                LastBoundDescriptorSetsBufferInfo[set].shouldReadBuffers = true
```

Comment:
Perhaps it is too much of a corner case to worry about, but if I bind descriptor sets, and then never make a draw/dispatch call that could use those descriptor sets, then I think the buffer shouldn't actually be read.

Reply:
This 'shouldReadBuffers' means their memory will be read the 'next time' they are used, so if they are never used in draw/dispatch, they will not be read. Anyway, as this comes with the optimization to minimize the number of accesses to descriptor sets, they will be reverted all together.